samples/client/openapi_features/security/python/src/this_package/configurations/api_configuration.py

```diff
@@ -96,6 +100,7 @@ class ApiConfiguration(object):
     :param security_index_info: path to security_index information
     :param server_info: the servers that can be used to make endpoint calls
     :param server_index_info: index to servers configuration
+    :param oauth_server_client_info: the oauth client_id and client_secret info
```

Comment:
Please update the comment here. `# the oauth client used for a specific oauth server` the current comment is incorrect